### PR TITLE
Playlist

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files to check
`src/components/modal/Modal.jsx`
`src/components/playlistcard/Playlistcard.jsx`
`src/context/playlist-context.js`
`src/page/playlist/Playlist.jsx`
`src/reducer/playlist-reducer.js`

### thank you for reviewing my pr  :)

### CODE REVIEW

**Review by arjun-computer-geek — APPROVED** [review]
Looks cleaner. Nice looking code. 🔥🔥🔥 @Developers-codz

File: src/components/modal/modal.module.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  min-width: 20rem;
+  max-height: 17rem;
+  z-index: 3;
+  background-color: var(--COLOR-BACKGROUND-LIGHTER);
+  border-radius: 20px;
+  color: white;
```
Comment (MuskanChhatrasal) [inline src/components/modal/modal.module.css:11]: Can predefine the color in root file.

**Review by MuskanChhatrasal — APPROVED** [review]
Great work, clean code. Nothing much to suggest just define all the color in root file. Else everything is good to merge.

File: src/components/modal/modal.module.css (hunk 1 above)
Comment (Developers-codz) [reply src/components/modal/modal.module.css:11]: Okay

**Pull request merged**